openml-lightgbm/pom.xml

```
@@ -24,6 +24,7 @@
 
     <properties>
         <lightgbmlib.version>2.3.150</lightgbmlib.version>
+        <make.lightgbmlib.version>2.3.1</make.lightgbmlib.version>
```

Use explicit versioning so we can replace by any commit in the future.
```suggestion
        <make.lightgbmlib.version>v2.3.1</make.lightgbmlib.version>
```